modules/deploy/pages/deployment-option/cloud/monitor-cloud.adoc

```diff
@@ -0,0 +1,71 @@
+= Monitor Redpanda Cloud
+:description: Learn how to integrate the Redpanda Cloud metrics endpoint to monitor the health of your Dedicated or BYOC clusters.
+:page-cloud: true
+:env-cloud: true
+
+You can configure monitoring on your Redpanda Dedicated or BYOC cluster to maintain system health and optimize performance. Redpanda exports metrics from the xref:reference:public-metrics-reference.adoc[`/public_metrics`] Prometheus endpoint through the Admin API port 9644. 
```

Comment:
Slight modification:

> Redpanda exports metrics from the `/public_metrics` Prometheus endpoint throught the Admin API port 9644

In cloud, we provide a straight https endpoint and don't expose the fact that this is hitting a loadbalancer and 9644 on the backend.  e.g.:

`https://console-<id>.<clusterid>.fmc.prd.cloud.redpanda.com/api/cloud/prometheus/public_metrics`

As well, this endpoint collects all the `/public_metrics` and kafka connect metrics that this cluster generates so you only have to scrape once (which is somewhat different to how this works in self-hosted).

Maybe stating something like:

> Redpanda Cloud exports metrics for all brokers and Kafka Connect workers from a single Prometheus endpoint. This endpoint can be found in the Control Plane user interface for your Redpanda cluster.

Reply:
@hcoyote Please check this [revised intro](https://deploy-preview-352--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/monitor-cloud/). We don't yet clearly document the difference in the UI between control plane and data plane, but I have a separate PR for it that I'll soon ask you to please look at!

Reply:
LGTM